### PR TITLE
feat(fa): Display municipality logo in the applciation

### PR DESCRIPTION
**Pull request by davidpxn**
# ... 👆

https://app.asana.com/0/1202167941440767/1202412116438909

## What

This a first step solution to be able to have the muncipality logo displayed in the Application System. We wanted to have the logo on the sidebar but that is currently not possible (that feature is on the backlog of Sendriráðið). In the meantime we will add the logo at the bottom of the pages instead.

- Create a higher order component "withLogo"
- Use "withLogo" to add the municipality logo to relevant pages.
- Add all logo svg files to project

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/financial-aid/src/fields/Logo/Logo.css.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { theme } from '@island.is/island-ui/theme'
+
+export const logo = style({
+  width: '50%',
+  '@media': {
+    [`screen and (min-width: ${theme.breakpoints.md}px)`]: {
+      width: '33%',
+    },
+  },
+})
```
Comment (MargretFinnboga) [inline libs/application/templates/financial-aid/src/fields/Logo/Logo.css.ts:11]: the styles on the logo are weird imo
Why is it not in the middle of the screen?

**Review by MargretFinnboga — COMMENTED** [review]
I have two pælingar
1. Does the logo have to be on every screen? Is it not enough to have it on the first one and last one maybe? 
2. The placement on the logo is a bit off, I would like to see it in the middle

File: libs/application/templates/financial-aid/src/fields/Logo/Logo.css.ts (hunk 1 above)
Comment (davidpxn) [reply libs/application/templates/financial-aid/src/fields/Logo/Logo.css.ts:11]: 1. They want the logo to be on every screen
2. They want the logo to be right aligned after we showed them other versions

**Review by MargretFinnboga — APPROVED** [review]
🙃

**Pull request merged**